Task14/src/test/java/bsu/tat/triangle/TriangleDefineTestj.java

```diff
@@ -0,0 +1,24 @@
+package bsu.tat.triangle;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+
+public class TriangleDefineTest {
+    TriangleDefine td;
+
+    @Test
+    public void testDefineType() throws Exception {
+        td = new TriangleDefine();
+    }
+
+    @Test
+    public void testDefineIsosceles() throws Exception {
+    }
```

Comment:
As you print out message instead of returning triangle type - you are not able to test it.